strr-web/interfaces/application-i.ts

```diff
@@ -55,9 +58,37 @@ export interface ApplicationDetailsI {
   documents?: DocumentUploadI[]
 }
 
+export interface PlatformApplicationDetailsI {
+  registrationType: RegistrationTypeE.PLATFORM,
+  completingParty?: ContactI,
+  platformRepresentatives: ContactI[],
+  businessDetails: {
+    legalName: string,
+    homeJurisdiction: string,
+    businessNumber?: string,
+    consumerProtectionBCLicenceNumber?: string,
+    noticeOfNonComplianceEmail: string,
+    noticeOfNonComplianceOptionalEmail?: string,
+    takeDownRequestEmail: string,
+    takeDownRequestOptionalEmail?: string,
+    mailingAddress: RegistrationAddressI & { locationDescription?: string },
+    registeredOfficeOrAttorneyForServiceDetails?: {
+      attorneyName: string,
+      mailingAddress: RegistrationAddressI & { locationDescription?: string }
+    }
+  },
+  platformDetails: {
+    brands: {
+      name: string,
+      website: string
+    }[],
+    listingSize: 'GREATER_THAN_THOUSAND' | 'LESS_THAN_THOUSAND'
+  }
+}
+
 export interface ApplicationI {
   header: ApplicationHeaderI
-  registration: ApplicationDetailsI
+  registration: ApplicationDetailsI | PlatformApplicationDetailsI
```

Comment:
It will cause issues, and should have one interface. We need to think how to fix it.

Reply:
Changes made